huaweicloud/utils/utils.go

```diff
@@ -283,6 +283,17 @@ func GetTimezoneCode() int {
 	return timezoneNum / 100
 }
 
+// UnitConver memory unit conversion, converting MB to GB.
+// For example 4096MB = 4GB
+func UnitConver(memory interface{}) int {
+	num, err := strconv.Atoi(memory.(string))
+	if err != nil {
+		fmt.Printf("convert string to int fail")
+	}
+	num /= 1024
+	return num
+}
+
```

Comment:
```suggestion
// ConvertMemoryUnit is a method that used to convert the memory unit.
// Parameters:
// + memory: Size of the memory in the current unit.
// + diffLevel: Difference level between units before and after conversion.
// diffLevel greater than 0 means that the unit is converted to a higher level, and vice versa to a lower level, e.g.
// the unit of memory input is MB, -2 means it is converted from MB to B, and 2 means it is converted from MB to TB.
func ConvertMemoryUnit(memory interface{}, diffLevel int) int {
	var memoryInt int
	switch memory.(type) {
	case int:
		memoryInt = memory.(int)
	case string:
		var err error
		memoryInt, err = strconv.Atoi(memory.(string))
		if err != nil {
			log.Printf("convert string value (%v) to int fail: %s", memory, err)
		}
	}

	if diffLevel >= 0 {
		return memoryInt / Power(1024, diffLevel)
	}
	return memoryInt * Power(1024, -diffLevel)
}
```
And you should support a function to calculate the power of a number.
```
func Power(base int, exponent int) int {
	if exponent == 0 {
		return 1
	}
	return base * Power(base, exponent-1)
}
```

Reply:
have changed